accession/utils.py

```diff
@@ -0,0 +1,62 @@
+from django.db.models import Count
+
+from accession import models
+
+
+class ModelNotFound(Exception):
+    def __init__(self, value):
```

Comment:
If we aren't doing anything unique in the body of this class, we could replace the content with a docstring and be done with it. We will still be able to pass a message.